### PR TITLE
Add alert to notify user of unverified routes

### PR DESCRIPTION
**Pull request by CloudNiner**
Also adds the wheelchair param to the Directions.get request in OverviewController

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 1]
```diff
@@ -108,6 +109,25 @@
             }
         }
 
+        /**
+         * Checks the geojson response to verify that each feature was audited
+         * @param  {geojson}  geojson Geojson object returned from Directions.get
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:114]: geojson param name here twice?

**Comment by flibbertigibbet** [comment]
Do we have a way to display warnings per route step?  I thought we were maybe going to change the line color for unaudited segments.

+1

File: src/nih_wayfinding/app/scripts/routing/directions-service.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:114]: heh no, geojson is the datatype and then also the param name.

**Comment by CloudNiner** [comment]
The line color does currently change depending on whether the route is audited or not. Blue if its audited, black if not. I snuck in that fix yesterday.

**Pull request merged**